### DISCUSSION TITLE
Welcome to PyEt Discussions!

### DISCUSSION THREAD

**Discussion by raoulcollenteur** [pyet-org/pyet#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by arjun231193** [comment]
I am always getting an error while calculating PET. I am new to coding and is not getting the error message correctly. Attached below is a snip. Please help.
![pyet error](https://user-images.githubusercontent.com/87454942/127414930-691fa396-49fa-4b94-a0d8-e6ae7252e8fb.png)

**Reply by mvremec** [reply]
Dear Arjun! 
Thank you for trying out pyet!
The error message you are getting is because the input for the Hargreaves function should 
 be a pandas time series, with a datetimeindex. 
A pandas time series is required because the function needs to compute extraterrestrial radiation
using the day of the year, which can is obtained from the pandas datetimeindex.

Hope this helps!
Best,
Matevz

**Comment by arjun231193** [comment]
Dear Concern

Thank you so much for the information. What if I want to calculate it for
multiple stations at a time? I am working on a gridded data set with each
grid having its own rainfall, tmax and tmin series. I have hundreds of
grids to work with. Can you please suggest to me a better way to use this
package in that case?
Thanks in advance.

Regards
*Arjun Kaushik*
Research Associate (Department Of Hydrology)
Indian Institute Of Technology Roorkee
Roorkee (Uttrakhand)- 247667
Mob No:- +919758212761, +4915738623359

**Reply by mvremec** [reply]
Dear Arjun,

at this point, you could loop through the gridded data set and use the needed 
pyet functions for the estimation of evaporation. 
For now, it is not yet possible to directly use gridded data sets with pyet (ndarray, xarray,...).
We will be working on implementing gridded data sets after the summer months.

Best,
Matevž

**Comment by arjun231193** [comment]
Dear Matevz

Thank you so much for the information.

Regards
*Arjun Kaushik*
Research Associate (Department Of Hydrology)
Indian Institute Of Technology Roorkee
Roorkee (Uttrakhand)- 247667
Mob No:- +919758212761, +4915738623359
